src/simulations/results/results_plotting_utils.jl

```diff
@@ -3,16 +3,7 @@ export plot_glacier
 function plot_glacier_heatmaps(results, variables, title_mapping)
 
     # Dictionary of variable-specific colormaps
-    colormap_mapping = Dict(
```

Comment:
Where did this go?

Reply:
I noticed that this color_mapping is unnecessary because I already defined the colormaps for specific variables in the "hub" function plot_glacier(), so now I am just making sure these colormaps are extracted from there.